README.md

```diff
@@ -7,10 +7,19 @@
 After you install Homebrew, run the following command:
 
 ```
-brew tap homebrew-community/mint
+brew tap mint-lang/homebrew-mint-lang
```

Comment:
```suggestion
brew tap mint-lang/mint-lang
```

Reply:
I would prefer this as well but, the repo name needs to start with `homebrew` unfortunately.

Reply:
`brew tap mint-lang/mint-lang` clones `mint-lang/homebrew-mint-lang`

Reply:
Nice, didn't know about that :+1: